app/assets/fonts/openproject_icon/src/resizer-vertical-lines.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.4, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="16px" height="16px" viewBox="0 0 16 16" enable-background="new 0 0 16 16" xml:space="preserve">

```

Comment:
Hey @HDinger, the XML look pretty different from all the other icons. It might be no problem. However, in the past I added an icon with a viewport that was wider than the typical ```<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">``` and all icons in the font were misplaced by some pixels.